<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add</class>
 <widget class="QDialog" name="Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="radioButton_man">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>男</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_woman">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>女</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>年龄：</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ADD">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>500</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>添加学生信息</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>41</y>
     <width>202</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="Add_nameEdit">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Add_idEdit">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Add_classEdit">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>40</y>
     <width>62</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>姓名：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>工号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>班级：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>152</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>语文成绩：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>数学成绩：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>202</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLineEdit" name="Add_chineseEdit">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="Add_mathEdit">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
